packages/store/src/symbols.ts

```diff
@@ -44,6 +44,12 @@ export class NgxsConfig {
   }
 }
 
+export interface ActionDef {
```

Comment:
you could also do `export type ActionDef<T = any> = Type<T> & { type: string };`

Reply:
true we could

Reply:
@deebloo `store/src/store.ts` uses `Type` from Angular lib, but it's up to you :wink:

Reply:
I think we should decrease the angular dependency, in order to someday make the library available also for other libraries/frameworks.

Reply:
@eranshmil hardly

Reply:
All the angular stuff should be removed from the core and moved to a plugin.